producer.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+# thoth-graph-refresh-job-producer
+# Copyright(C) 2020 Sai Sankar Gochhayat, Fridolin Pokorny
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Refresh data stored in the graph database.
+
+Produces messages regarding package depending on the component in which is used,
+which are picked up by investigator to schedule solvers.
+"""
+
+import logging
+import random
+import os
+import asyncio
+
+from thoth.common import init_logging
+from thoth.common import OpenShift
+from prometheus_client import CollectorRegistry, Gauge
+
+from thoth.messaging import MessageBase
+from thoth.messaging.unresolved_package import UnresolvedPackageMessage
+from thoth.messaging.unrevsolved_package import UnrevsolvedPackageMessage
+from thoth.common import __version__ as __common__version__
+from thoth.storages import __version__ as __storage__version__
+from thoth.storages import GraphDatabase
+
```

Comment:
`from thoth.messaging important __version as __messaging_version__`

and let's add it to `__service_version__` so that we can see which versions of thoth libraries are used